quic_transport/sdk/impl/quic_transport_owt_client_impl.cc

```diff
@@ -167,6 +178,17 @@ const char* QuicTransportOWTClientImpl::Id() {
 
 }
 
+void QuicTransportOWTClientImpl::CloseStreamOnCurrentThread(uint32_t id) {
+  printf("QuicTransportOWTClientImpl::CloseStreamOnCurrentThread close stream:%d\n", id);
+  session_->ResetStream(id, quic::QUIC_STREAM_CANCELLED);
```

Comment:
Closing a QUIC stream with RESET may end up with data loss. Consider sending a STREAM frame with FIN set.

Reply:
Please check this again. I would suggest to use FIN instead of RESET.